api/nakadi-event-bus-api.yaml

```diff
@@ -34,22 +99,252 @@ paths:
       summary: Get monitoring metrics
       responses:
         '200':
-          description: Metrics data
+          description: Ok
           schema:
             $ref: '#/definitions/Metrics'
         '401':
-          description: User is not authenticated
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
           schema:
             $ref: '#/definitions/Problem'
         '503':
           description: Not available
           schema:
             $ref: '#/definitions/Problem'
+  /event-types:
+    get:
+      tags:
+        - schema-registry-api
+      description: Returns a list of all registered `EventType`s
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/EventType'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+    post:
+      tags:
+        - schema-registry-api
+      description: |
+        Creates a new `EventType`. Upon creation the name MUST be unique. Failing to do so will fail the
+        request with a 422 Unprocessable entity.
+
+        **Implementation note:** The creation of an EventType implicitly creates the structures in the backing
+        messaging implementation needed for the reception and persistency of the Events. Considering that at
+        this time only Kafka is used, this corresponds to the creation of a Topic. If so desired, clients can
+        interact directly with the topic using the low level API (for existing restrictions see the
+        corresponding methods on the topic-api).
```

Comment:
At first we'll not be supporting any refined way of managing this. 
There is a field "cardinality" that is to be provided on creation of the EventType that will work as a hint of the load this EventType/Topic might face and will influence this. 

Just a couple notes about the possibilities that we "lose" by not having 2 separate steps, and why they are not problematic:
- we're being consistent with the hiding of topics and putting the EventType as the first class citizen;
- if one wants a "raw topic" where he can put anything in or use in the "low level way", simply as point to point communication, he simply needs to create an EventType that does not specify anything related to validation or any other enforcement. The behaviour is the same at the end.